.gitignore

```diff
@@ -17,6 +17,9 @@ yarn.log
 # Don't commit ESLint error reports
 eslint_report.htm
 
+# Ignore built files
+dist/**/*
```

Comment:
Same as above? I'd like to have as less additional stuff to do as possible, so I'd rather have the build vomit everything into the right directory directly (we could create two directives for debug with types, and for release only with sourcemaps)